app/server/controllers/UserController.js

```diff
@@ -736,4 +736,116 @@ UserController.getStats = function(callback){
   return callback(null, Stats.getUserStats());
 };
 
+UserController.addUserAcceptedQueue = function(id, callback){
+  User.findOneAndUpdate({
+    _id: id,
+    'status.admitted': false
+  },{
+    $set: {
+      'status.queued': Date.now(),
+    }
+  },
+  function(err, user){
+    if(err || !user){
+      return callback(err);
+    }
+    if(user.status.queued){
+      return callback({
+        message: `User id:${id}, email:${user.email} already to acceptance queue`
+      });
+    } else {
+      return callback(null, {
+        message: `User id:${id}, email:${user.email} added to acceptance queue`
+      });
+    }
+  });
+};
+
+UserController.removeUserAcceptedQueue = function(id, callback){
+  User.findOneAndUpdate({
+    _id: id
+  },{
+    $set: {
+      'status.queued': null
+    }
+  },
+  function(err, user){
```

Comment:
Please use the default "callback" variable that is passed in from the function. I can validate on the front end whether or not a user has been accepted into the queue

Reply:
Avoid using the custom messages since that is throwing off how I am handling the data on the front end